.gitignore

```diff
@@ -127,3 +127,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# Redacted Images
```

Comment:
This file is boilerplate. Could you put the additional custom rules at the top of the file, where it's more visible and obvious that we've added it?

Otherwise, LGTM.